js/generator.js

```diff
@@ -64,9 +70,71 @@ class Generator {
       case 4:
         event.target.setAttribute('fill', '#196127');
         break;
+      case 5:
+        event.target.setAttribute('fill', '#ebedf0');
+        this.daysActivity[index] = 0;
+        break;
     }
   }
 
+  toneUp = (event) => {
+    for (let i = 0; i < this.days.length; i++) {
```

Comment:
You don't need this `for loop` and you can add only the following code into the next `for loop` to do same:

```
      this.daysActivity[i] = this.daysActivity[i] ? this.daysActivity[i] + 1 : 1;
```